MVP Module/Source/Feed UI/Composers/FeedViewAdapter.swift

```diff
@@ -5,7 +5,7 @@
 import UIKit
 import FeedFeature
 
-final class FeedViewAdapter: FeedView {
+final class FeedViewAdapter: FeedView, FeedErrorView {
```

Comment:
There's no need for making the `FeedViewAdapter` implement the `FeedErrorView` protocol, since there's nothing in error logic that needs adapting. It's easier to make the `FeedViewController` conform to `FeedErrorView`.
```suggestion
final class FeedViewAdapter: FeedView {
```

Reply:
edited, if we are using in presenter FeedErrorView protocol, we need that protocol in  adapter also

Reply:
done